src/components/Activists/Member.jsx

```diff
@@ -49,85 +61,190 @@ const Member = ({ member, handleClose }) => {
   if (!member) return null
 
   return (
-    <Box sx={{ width: '32rem', height: '100%' }} role="slide-over" data-cy="contact-member-detail">
-      <Box sx={{ overflowY: 'scroll', height: '100%', bgcolor: 'whiteCorner' }}>
-        <Box
-          sx={{
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'space-between',
-            p: 2,
-            color: 'colors.gray.400',
-          }}
-        >
-          <Box display="flex" flexDirection="column" className="space-y-2">
-            <Typography variant="h5" sx={{ color: 'colors.gray.800' }}>
-              {member.firstname} {member.lastname}
-            </Typography>
+    <Box
+      sx={{
+        width: '34rem',
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        bgcolor: 'whiteCorner',
+        justifyContent: 'space-between',
+      }}
+      role="slide-over"
+      data-cy="contact-member-detail"
+    >
+      <Box
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+          py: 2,
+          px: 3,
+          color: 'colors.gray.400',
+        }}
+      >
+        <Box display="flex" flexDirection="column" className="space-y-2">
+          <Typography variant="h5" sx={{ color: 'colors.gray.800' }}>
+            Fiche du militant
+          </Typography>
+        </Box>
+        <IconButton edge="start" color="inherit" onClick={handleClose} aria-label="close">
+          <CloseIcon />
+        </IconButton>
+      </Box>
+      <Box
+        sx={{
+          position: 'relative',
+          flex: '1 1 0%',
+          overflow: 'hidden',
+          overflowY: 'scroll',
+          height: '100%',
+        }}
+        className="scrolling-bar"
+      >
+        <Box sx={{ position: 'absolute', top: 0, left: 0, right: 0, height: '90px', bgcolor: 'colors.blue.900' }}>
+          <div className="heading-banner"></div>
+        </Box>
+        <Box sx={{ px: 3, pt: 8 }}>
+          <Box sx={{ display: 'flex', alignItems: 'flex-end' }} className="space-x-2">
+            <Avatar
+              sx={{
+                width: 60,
+                height: 60,
+                bgcolor: 'colors.blue.500',
+                border: '2px solid',
+                borderColor: 'whiteCorner',
+              }}
+            >
+              {member.firstname.split(' ')[0][0]} {member.lastname.split(' ')[0][0]}
```

Comment:
pourquoi pas juste :

```suggestion
              {member.firstname[0]} {member.lastname[0]}
```